stress/common/src/main/java/alluxio/stress/fuse/FuseIOParameters.java

```diff
@@ -48,9 +45,14 @@
           description = "The buffer size for IO operations. (1k, 16k, etc.)")
   public String mBufferSize = "64k";
 
-  @Parameter(names = {"--num-files"},
-      description = "The number of files for the test")
-  public int mNumFiles = 1000;
+  @Parameter(names = {"--num-files-per-dir"},
+      description = "The number of files per directory for the test")
+  public int mNumFilesPerDir = 1000;
+
+  @Parameter(names = {"--num-dirs"},
+      description = "The number of directories that the files will be evenly distributed into."
+          + "It is suggested to be a multiple of the number of threads.")
```

Comment:
The number of directories should be bigger or equal to the thread number, and it's suggested to be a multiple of the number of threads.